### PR TITLE
docs: re-arrange csw docs and examples

### PR DESCRIPTION
**Pull request by shimoncohen**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✖                                                                        |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✔                                                                        |
| Tests added     | ✖                                                                        |
| Chore            | ✔                                                                       |

### CODE REVIEW

**Review by asafMasa — CHANGES_REQUESTED** [review]
image
Doesn't work

**Comment by shimoncohen** [comment]
@asafMasa I made the requested changes, can you please review again?

File: docs/MapColonies/DEM/Services/catalog/profile_v1_0.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ tags:
 | mc:SRSName | text | name of reference system |
 | mc:verticalDatum | enum  | **Valid Values**: <br/> WGS 1984 / WGS 1972 / Pulkovo 1942 / Palestine 1923 / MSL Height / Israel / ED 1950 IDF |
 | mc:region | text | sector / countries <br/> comma separated list |
-| mc:classification | enum  | product classification / confidentiality <br /> [Classification values](/classified/dem/classification_table.md) |
+| mc:classification | enum  | product classification / confidentiality <br /> Classification values |
```
[the remark this answers is not in the document]
Comment (shimoncohen) [reply docs/MapColonies/DEM/Services/catalog/profile_v1_0.md]: It's ok, I will make sure changes are sustained.

File: docs/MapColonies/Raster/Guides/raster-getting-started.mdx [hunk 2]
```diff
@@ -49,7 +49,7 @@ flowchart LR
 ```
 
 ## Query Catalog Service (Step 1)
-Query **RASTER CSW catalog** service to find item(s) according to desired filter [example are here](/docs/ogc/protocols/examples/ogc-csw-examples).
+Query **RASTER CSW catalog** service to find item(s) according to desired filter [example are here](/docs/ogc/protocols/ogc-csw/ogc-csw-examples).
```
Comment (asafMasa) [inline docs/MapColonies/Raster/Guides/raster-getting-started.mdx:52]: image
doesn't work as expected

File: docs/MapColonies/Raster/services/catalog/README.md [hunk 3]
```diff
@@ -12,7 +12,7 @@ tags:
 ---
 
 # Catalog Service 🗃️
- [Catalog Service for the Web (CSW)](/docs/ogc/protocols/ogc-csw) is a standard for exposing a catalog of geospatial records in XML on the Internet (over HTTP). The catalog is made up of records that describe geospatial data, geospatial services (e.g. [WMTS](/docs/ogc/protocols/ogc-wmts)), and related resources.
+ [Catalog Service for the Web (CSW)](/docs/ogc/protocols/ogc-csw/ogc-csw) is a standard for exposing a catalog of geospatial records in XML on the Internet (over HTTP). The catalog is made up of records that describe geospatial data, geospatial services (e.g. [WMTS](/docs/ogc/protocols/ogc-wmts)), and related resources.
```
Comment (asafMasa) [inline docs/MapColonies/Raster/services/catalog/README.md:15]: image
why this property has a different color from all of the rest?

File: docs/ogc/protocols/ogc-csw/README.md [hunk 4]
```diff
@@ -17,11 +17,11 @@ tags:
 Overview - source
 The CSW service exposes the metadata records in XML format using the [OGC CSW protocol](https://docs.pycsw.org/en/latest/introduction.html#csw-support) 🌐
 
-[CSW Registry Service (Offline)](../../../static/pdfs/manuals/07-110r4_CSW-ebRIM_Registry_Service_-_Part_1_ebRIM_profile_of_CSW_v1.0.1.pdf "download") 📚
+[CSW Registry Service (Offline)](../../../../static/pdfs/manuals/07-110r4_CSW-ebRIM_Registry_Service_-_Part_1_ebRIM_profile_of_CSW_v1.0.1.pdf "download") 📚
```
Comment (asafMasa) [inline docs/ogc/protocols/ogc-csw/README.md:20]: image

Do we want to link our available  CSW profiles here?

**Review by asafMasa — CHANGES_REQUESTED** [review]

File: docs/ogc/protocols/ogc-csw/README.md (hunk 4 above)
Comment (shimoncohen) [reply docs/ogc/protocols/ogc-csw/README.md:20]: Added.

File: docs/MapColonies/Raster/services/catalog/README.md (hunk 3 above)
Comment (shimoncohen) [reply docs/MapColonies/Raster/services/catalog/README.md:15]: Fixed.

File: docs/MapColonies/Raster/Guides/raster-getting-started.mdx (hunk 2 above)
Comment (shimoncohen) [reply docs/MapColonies/Raster/Guides/raster-getting-started.mdx:52]: Fixed.

**Review by asafMasa — CHANGES_REQUESTED** [review]
Looking Good 😎

File: docs/MapColonies/Raster/services/catalog/profile_v1_0.md [hunk 5]
```diff
@@ -21,7 +21,7 @@ tags:
 | mc:productId | text | unique external product id |
 | mc:productName | text | the product name |
 | mc:productVersion | text | the product version |
-| [mc:productType](#productType) | enum  | **Valid Values**: <br/> OrthophotoBest / Orthophoto / OrthophotoHistory / RasterMapBest / RasterMap / RasterAidBest / RasterAid / RasterVector / RasterVectorBest |
+| mc:productType | enum  | **Valid Values**: <br/> OrthophotoBest / Orthophoto / OrthophotoHistory / RasterMapBest / RasterMap / RasterAidBest / RasterAid / RasterVector / RasterVectorBest |
```
[the remark this answers is not in the document]
Comment (shimoncohen) [reply docs/MapColonies/Raster/services/catalog/profile_v1_0.md:24]: Why would you write all of these different comments in one thread???

[the remark this answers is not in the document]
Comment (asafMasa) [reply docs/MapColonies/Raster/services/catalog/profile_v1_0.md:24]: i've opened relevent issues

**Review by asafMasa — DISMISSED** [review]
Great Job!
It's looking good 😎

**Review by alebinson — APPROVED** [review]
Looks good

**Review by netanelC — APPROVED** [review]

**Pull request merged**